sites/public/src/tsx_content/get-started-cards.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react"
```

Comment:
These `tsx_content` files are filled with lorem ipsum and are just placeholders until we add actual content.

(also this CL is a lot of lines of code, but ~5-600 of it is just lorem ipsum from these files)